src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl

```diff
@@ -1800,7 +1803,8 @@
           "description",
           "status",
           "paymentTypeCode",
-          "ranges"
+          "ranges",
+          "methodManagement"
```

Comment:
a required field for the api currently used in prod by checkout, is this PR backwards compatible? does it require a definition of a release strategy?

Reply:
Momentarily we could avoid updating the openapi of **ecommerce for checkout** by ensuring backward compatibility. What do you think ?